controllers/sriovnetworknodepolicy_controller.go

```diff
@@ -272,6 +273,13 @@ func (r *SriovNetworkNodePolicyReconciler) syncAllSriovNetworkNodeStates(ctx con
 		ns.Name = node.Name
 		ns.Namespace = vars.Namespace
 		j, _ := json.Marshal(ns)
+		netPoolConfig, _, err := utils.FindNodePoolConfig(ctx, &node, r.Client)
```

Comment:
just a general todo here we should have in memory map I think for this

Reply:
could you please elaborate on this?